### PR TITLE
API updates for 0.2.0-preview

### PR DESCRIPTION
**Pull request by jennyf19**
TODO:
- [x] Review migration aid + experience
- [x] Update to latest Microsoft Identity Web
- [x] Test all samples

cc: @jmprieur

### CODE REVIEW

File: 2-WebApp-graph-user/2-2-TokenCache/README-incremental-instructions.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ public void ConfigureServices(IServiceCollection services)
     // Token acquisition service based on MSAL.NET 
     // and the Sql server based token cache implementation
     services.AddMicrosoftIdentityPlatformAuthentication(Configuration)
```
Comment (jmprieur) [inline 2-WebApp-graph-user/2-2-TokenCache/README-incremental-instructions.md:99]:
```suggestion
    services.AddMicrosoftWebAppAuthentication(Configuration)
```

Comment (jennyf19) [reply 2-WebApp-graph-user/2-2-TokenCache/README-incremental-instructions.md]: nice catch! thank you!

File: 2-WebApp-graph-user/2-3-Multi-Tenant/Startup.cs [hunk 2]
```diff
@@ -85,11 +86,9 @@ public void ConfigureServices(IServiceCollection services)
                     };
                 }, options =>
```
Comment (pmaytak) [inline 2-WebApp-graph-user/2-3-Multi-Tenant/Startup.cs:87]: Remove cookie action after PR [309](https://github.com/AzureAD/microsoft-identity-web/pull/309) is merged.

File: 4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs [hunk 3]
```diff
@@ -43,17 +43,8 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddOptions();
 
-            services.AddSignIn(Configuration);
-
-            // This is required to be instantiated before the OpenIdConnectOptions starts getting configured.
-            // By default, the claims mapping will map claim names in the old format to accommodate older SAML applications.
-            // 'http://schemas.microsoft.com/ws/2008/06/identity/claims/role' instead of 'roles'
-            // This flag ensures that the ClaimsIdentity claims collection will be built from the claims in the token
-            // JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
```
Comment (pmaytak) [inline 4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs:52]: Hmm, wonder if we need to leave this comment?

File: 4-WebApp-your-API/4-2-B2C/Client/Startup.cs [hunk 4]
```diff
@@ -42,17 +42,8 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddOptions();
 
-            services.AddSignIn(Configuration, "AzureAdB2C");
-
-            // This is required to be instantiated before the OpenIdConnectOptions starts getting configured.
-            // By default, the claims mapping will map claim names in the old format to accommodate older SAML applications.
-            // 'http://schemas.microsoft.com/ws/2008/06/identity/claims/role' instead of 'roles'
-            // This flag ensures that the ClaimsIdentity claims collection will be built from the claims in the token
-            // JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
```
Comment (pmaytak) [inline 4-WebApp-your-API/4-2-B2C/Client/Startup.cs:51]: Same as above.

File: 5-WebApp-AuthZ/5-2-Groups/Startup.cs [hunk 5]
```diff
@@ -36,19 +36,9 @@ public void ConfigureServices(IServiceCollection services)
...
-            services.AddSignIn(Configuration);
-
-            // If you want to use group ids/names in the Authorize attribute then uncomment the following lines:
-            //services.Configure<OpenIdConnectOptions>(options => 
-            //{
-            //    // Uncomment the following lines code instruct the asp.net core middleware to use the data in the "groups" claim in the [Authorize] attribute and for User.IsInrole()
-            //    // See https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles for more info.
-            //    // Use the groups claim for populating roles
-            //    options.TokenValidationParameters.RoleClaimType = "groups";
-            //});
```
Comment (pmaytak) [inline 5-WebApp-AuthZ/5-2-Groups/Startup.cs:48]: Need to leave this comment,

File: 4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs (hunk 3 above)
Comment (jmprieur) [reply 4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs:52]: It depends on the version of Wilson. ?

File: 5-WebApp-AuthZ/5-2-Groups/Startup.cs (hunk 5 above)
Comment (jmprieur) [reply 5-WebApp-AuthZ/5-2-Groups/Startup.cs:48]: yes this one should remain.

File: 4-WebApp-your-API/4-2-B2C/Client/Startup.cs (hunk 4 above)
Comment (jmprieur) [reply 4-WebApp-your-API/4-2-B2C/Client/Startup.cs:51]: we need to test it.

**Comment by jmprieur** [comment]
@kalyankrishna1 @TiagoBrenck @Shama-K 
you might want to have a look at the PR.

File: 5-WebApp-AuthZ/5-2-Groups/Startup.cs (hunk 5 above)
Comment (jennyf19) [reply 5-WebApp-AuthZ/5-2-Groups/Startup.cs]: done.

**Pull request merged**